### ISSUE TITLE
Potential issue with how i_contact_neg/pos are calculated

### ISSUE THREAD

**Issue by hello-binit** [hello-robot/stretch_body#109]
https://github.com/hello-robot/stretch_body/blob/1bb13daa92216b624846105eb0e142eca40be2b2/body/stretch_body/arm.py#L228-L238

When calculating `i_contact_neg/pos`, the arguments are passed through `translate_force_to_motor_current` before being `max/min` compared. I think the `max/min` comparison should happen first.

The snippet above is from arm.move_by, but the same comparison happens in many locations.

**Comment by aedsinger** [comment]
Good catch. Because forces are higher than currents this bug effectively turned off the clipping to contact_thresh_max_n. Fixed in PR https://github.com/hello-robot/stretch_body/pull/116
